racingcar-java/src/test/java/mission02/CalculatorTest.java

```diff
@@ -0,0 +1,59 @@
+package mission02;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+/**
+ * 계산기테스트이므로, input과 output에 대해서 체크만 한다.
+ */
+
+public class CalculatorTest {
+
+
+    @DisplayName("올바른 식의 형태 - 정상 값")
+    @ParameterizedTest
+    @CsvSource({"1 + 2 + 3 * 4, 24", "2 * 3 / 3 - 1, 1", "2 * 3 * 4 - 4, 20"})
+    void 문자열에_올바른_식을_넣으면_의도대로_계산된_값이_나온다(String input, int output){
+        // when
+        final Calculator calculator = new Calculator(input);
+        final int result = calculator.getCalculatorValue();
```

Comment:
```suggestion
        // given
        final Calculator calculator = new Calculator(input);
        
        // when
        final int result = calculator.getCalculatorValue();
```
앞서 이야기한대로 BDD 방법에 대해서 조금 더 고려해볼 필요가 있을 것 같네요